modules/unsupported/swing/src/main/java/org/geotools/swing/tool/InfoToolHelper.java

```diff
@@ -170,8 +170,9 @@ protected MathTransform getContentToLayerTransform() {
                         }
                     }
                 }
+            }
 
-            } else {
+            if (transform == null && !transformFailed) {
```

Comment:
This changes the behaviour such that the following comment is no longer valid.  That should probably be removed. 

Possibly we should be logging a warning in this case too - the user asks for a content to layer transformation, but we gave back a default (identity?) case because values were not set correctly.

Reply:
@kouta-kun, can you address @bradh feedback? We don't merge PRs that failed review.